about.html

```diff
@@ -52,17 +69,12 @@ <h1>About Me</h1>
             <div class="work-experience">
                 <h1>Work Experience</h1>
                 <div class="function-work">
-                    <h2>Junior Web Developer</h2>
+                    <h2>Creation of own projects</h2>
```

Comment:
Adicionar texto novo
```suggestion
                    <h2>My Learning Strategy</h2>
```